README.md

```diff
@@ -5,6 +5,26 @@ repository at <https://git.kitenet.net/index.cgi/git-annex.git/>.  The remote
 The local `master` branch in this repository is only used to store CI
 configuration and associated scripts.
 
+## Submitting Patches
+
+Patches to the git-annex source code can be added by opening a pull request
+against `master` that adds a Git patch file with a `.patch` file extension to
+the `patches/` directory; the CI will then build git-annex with this patch (and
+any others currently in `patches/`) applied.  If the patch fails to apply or
+appears to have already been applied upstream, the builds will fail.
+
+It is recommended that patches be given names of the form
+`YYYYMMDD-{commit}-{brief_description}.patch`, where `{commit}` is the short
+hash of the git-annex source code commit against which the patch was made.
```

Comment:
this format seems to be not providing strict ordering for the common "base" (`YYYYMMDD-{commit}-`)... anyways -- I do not care much since we unlikely to carry patches here in general, we just want to describe/state somewhere if there is ordering done